### ISSUE TITLE
Microsoft.CommonLanguageServerProtocol.Framework contains a proprietary License.txt

### ISSUE THREAD

**Issue by omajid** [dotnet/source-build#3530]
https://github.com/dotnet/dotnet/blob/826dcac6ef3a34e4e41042fb1112995fe1f21609/src/roslyn/src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/License.txt says:

```
1. INSTALLATION AND USE RIGHTS.
You may install and use any number of copies of the software to use solely with
Visual Studio Community
Visual Studio Professional
Visual Studio Enterprise
Visual Studio Code
...
In addition, you may not
...
* share, publish, rent, or lease the software; or
...
```

Should we remove Microsoft.CommonLanguageServerProtocol.Framework and subdirectories from the VMR?

**Comment by MichaelSimons** [comment]
[Triage] @oleksandr-didyk - Can you take this one?  TIA.

**Comment by oleksandr-didyk** [comment]
The package has quite a few references throughout `roslyn`, not sure we can just baseline it from the VMR (just in case tried it - the build fails miserably).

I'll contact the maintainers and consult them on the best approach here

**Comment by oleksandr-didyk** [comment]
https://github.com/dotnet/roslyn/issues/68781

**Comment by oleksandr-didyk** [comment]
Tracking issue on `roslyn` side - https://github.com/dotnet/roslyn/issues/68463

From conversations with the `roslyn` team we know that they are looking into possibly updating the license for the package in question and would let us know as soon as they have something concrete

**Comment by sharwell** [comment]
I believe this was fixed by https://github.com/dotnet/roslyn/pull/68942

**Comment by oleksandr-didyk** [comment]
> I believe this was fixed by [dotnet/roslyn#68942](https://github.com/dotnet/roslyn/pull/68942)

Great to hear, thank you for letting us now!

**Comment by oleksandr-didyk** [comment]
Closing as the change with removing the license has made it into the VMR with https://github.com/dotnet/dotnet/commit/7c263ce947cb4e46fee5f13e9ece6b93038d09af

**Comment by beccamc** [comment]
@omajid Just to confirm, the only issue was with this Microsoft.CommonLanguageServerProtocol.Framework license (which has been corrected). You aren't seeing any other licensing issues, correct?

**Comment by omajid** [comment]
> You aren't seeing any other licensing issues, correct?

Yes. I ran a scan on dotnet/roslyn and I don't see our tooling complain about any other files in roslyn.
